recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "16.0.0" %}
+{% set version = "16.2.0" %}
 
 package:
   name: path
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/path/path-{{ version }}.tar.gz
-  sha256: 48ddde18c6a15d40233bffeca882b6297a3e3a058903f1d4b52671cb8efd4018
+  sha256: 2de925e8d421f93bcea80d511b81accfb6a7e6b249afa4a5559557b0cf817097
 
 build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - wheel
     - setuptools >=42
```

Comment:
setuptools should be `setuptools >=56`, see https://github.com/jaraco/path/blob/983f8254e7767a3b13f54633157d7f24ad106649/pyproject.toml#L2

Reply:
setuptools was updated